dbt_common/events/logger.py

```diff
@@ -120,7 +122,11 @@ def create_line(self, msg: EventMsg) -> str:
     def write_line(self, msg: EventMsg):
         line = self.create_line(msg)
         if self._python_logger is not None:
-            send_to_logger(self._python_logger, msg.info.level, line)
+            if isinstance(msg.data, PrintEvent):
+                level = "error"
```

Comment:
I'm not following why this happens?

Reply:
This is to make sure the PrintEvent still makes it to stdout even when quiet is specified. Let me add a comment since it is not obvious

Reply:
Done